package.json

```diff
@@ -88,6 +88,7 @@
     "moment-timezone": "^0.5.41",
     "node-cache": "^5.1.2",
     "node-fetch": "^2.6.9",
+    "oauth-1.0a": "^2.2.6",
```

Comment:
Do we still need this ? If not, can we remove this ?!

Reply:
yes , we need this for generating OAuth header